onboard/cv/meson_options.txt

```diff
@@ -1,6 +1,9 @@
-option('tb_detection', type: 'boolean', value : true)
+option('ar_detection', type: 'boolean', value : true)
```

Comment:
**Nit:** As I mentioned above, change the grouping here

Reply:
Np. I see you're feeling nitty today.